src/backend/marsha/core/api.py

```diff
@@ -868,15 +868,16 @@ def post(self, request):
 
         """
         user = request.user
+        model = apps.get_model(app_label="core", model_name=resource)
```

Comment:
Maybe you should check that the resource is an allowed resource (think of "ceinture+bretelles")?

Reply:
regex are evil.